wherehows-web/app/components/dataset-confidential.js

```diff
@@ -70,15 +84,31 @@ export default Component.extend({
   }),
 
   /**
+   * Computed property that is dependent on all the keys in the datasetClassification map
+   *   Returns a new map of datasetClassificationKey: String-> Object.<Boolean|undefined,String>
+   * @type {Ember.computed}
+   */
+  datasetClassification: computed(`${datasetClassificationKey}.{${datasetClassifiersKeys.join(',')}}`, function() {
+    const sourceDatasetClassification = getWithDefault(this, datasetClassificationKey, {});
+
+    return Object.keys(datasetClassifiers).reduce((datasetClassification, classifier) => {
+      datasetClassification[classifier] = {
+        value: sourceDatasetClassification[classifier],
+        label: datasetClassifiers[classifier]
+      };
+
+      return datasetClassification;
+    }, {});
+  }),
+
+  /**
    * Creates a lookup table of fieldNames to classification
    *   Also, the expectation is that the association from fieldName -> classification
    *   is one-to-one hence no check to ensure a fieldName gets clobbered
    *   in the lookup assignment
    */
   fieldNameToClass: computed(
-    `${sourceClassificationKey}.confidential.[]`,
-    `${sourceClassificationKey}.limitedDistribution.[]`,
-    `${sourceClassificationKey}.highlyConfidential.[]`,
+    `${sourceClassificationKey}.{confidential,limitedDistribution,highlyConfidential}.[]`,
```

Comment:
Interesting, I haven't seen this syntax before for computed properties. Is this new?